### ISSUE TITLE
PySide6 v6.8.0 crashes on missing 'Element' class in QPainterPath

### ISSUE THREAD

**Issue by florisla** [thocoo/gamma-desk#34]
It seems a freshly installed gdesk[PySide6] either fails to start (qtpy not finding a Qt lib) or fails to show the image viewer.

Reverting to "PySide6==6.4.1" and "numpy<=2.0" resolves this.
I will look into it, this issue is here just to spread awareness.

Seen on Linux and Windows, Python versions 3.10 or 3.11.

**Comment by florisla** [comment]
The `qtpy` issue seems resolved.  But PySide6 v6.8.0 and v6.8.1 crash on `QPainterPath.Element` not being found.

However, the gdesk code does not use any 'Element' attribute.

I suspect this is a bug in PySide6 v6.8, which is still fresh (oct 10).  I can't find a bug report [here](https://bugreports.qt.io/browse/PYSIDE-2899?jql=project%20%3D%20PYSIDE) though.

`pip install PySide6<6.8` is enough to work around this.

```
ERR gdesk.core.stdinout 221:   File "/home/flo/tmp/venv/lib/python3.12/site-packages/gdesk/graphics/view.py", line 159, in pixelSize
ERR gdesk.core.stdinout 221:     p01 = tr.map(p0)
ERR gdesk.core.stdinout 221:           ^^^^^^^^^^
ERR gdesk.core.stdinout 221: AttributeError: type object 'PySide6.QtGui.QPainterPath' has no attribute 'Element'. Did you mean: 'elementAt'?
```

**Comment by florisla** [comment]
Fixed in 73483a24d0d1c85266e271c42da54a0d879b48fb.

**Comment by florisla** [comment]
The missing Element class is mentioned in this bug report: https://bugreports.qt.io/browse/PYSIDE-2898?jql=project%20%3D%20PYSIDE
